compiler-rt/lib/asan/asan_interceptors_memintrinsics.cpp

```diff
@@ -22,6 +22,14 @@
 
 using namespace __asan;
 
+// AIX does not intercept memcpy, so we have to use internal_memcpy.
+#if !SANITIZER_AIX
+#  define ASAN_MEMCPY_RETURN(to, from, size) return REAL(memcpy)(to, from, size)
```

Comment:
`#define ASAN_MEMCPY_RETURN(to, from, size) REAL(memcpy)(to, from, size)`

to keep return at use site